### PR TITLE
2. Layout 

### PR DESCRIPTION
**Pull request by dangvanhieu2**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+
+  def full_title page_title = ""
```
Comment (tranducquoc) [inline app/helpers/application_helper.rb]: def full_title page_title

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
+    if page_title.empty?
+      t"title_project"
```
Comment (tranducquoc) [inline app/helpers/application_helper.rb]: t "title_project"

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (dangvanhieu2) [inline app/helpers/application_helper.rb]: Đã fix

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (dangvanhieu2) [inline app/helpers/application_helper.rb]: Đã fix

**Pull request merged**